src/Populator/SamePropertyPopulator.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Neusta\ConverterBundle\Populator;
+
+/**
+ * @template S of object
+ * @template T of object
+ * @template C of object
+ * @implements Populator<S, T, C>
+ */
+class SamePropertyPopulator implements Populator
+{
+    public function __construct(
+        private string $propertyName
+    )
+    {
+    }
+
+    /**
+     * @inheritDoc
+     */
```

Comment:
I don't think we need this `@inheritDoc`, see: https://stackoverflow.com/a/26554272